### DISCUSSION TITLE
Customize axis labels?

### DISCUSSION THREAD

**Discussion by dgwyer** [observablehq/plot#965] [Q&A] [answered]
Again, I'm new to Observable plot so apologies if this is a trivial question but is there any scope to adjust the layout (location) and formatting (style) of the axis labels? I've searched around but all plot examples seem to have the same axis label types.

**Comment by Fil** [comment]
Please do not open _issues_ to get support; you're welcome to ask questions in the [discussions](https://github.com/observablehq/plot/discussions) section, or on the [user forum](https://talk.observablehq.com/).

**Reply by dgwyer** [reply]
Got it, thanks.

**Comment by dgwyer** [comment, marked as answer]
After reading the docs this can, of course, be done with the `labelAnchor` and `labelOffset` properties. Although it would be nice to have more `className` and `style` properties available for individual elements (marks, axes, labels, text etc.) rather than just for the global chart.

This would make it easy to customise different aspects of the chart. For example, currently I'm not sure how I'd style the axis tick numbers separately to the axis label text?

**Reply by Fil** [reply]
https://github.com/observablehq/plot/issues/6
